source/material_model/melt_simple.cc

```diff
@@ -301,6 +311,21 @@ namespace aspect
                                                * out.densities[i] / this->get_timestep();
                   else
                     out.reaction_terms[i][c] = 0.0;
+
+                  // fill reaction rate outputs if the model uses operator splitting
+                  if (this->get_parameters().use_operator_splitting)
+                    {
+                      if (reaction_rate_out != NULL)
```

Comment:
so if somebody forgets to add the outputs but uses operator splitting it should not be an error?